scripts/scil_tractprofiles.py

```diff
@@ -16,11 +16,14 @@
 
 from scilpy.io.streamlines import load_tractogram_with_reference
 from scilpy.io.image import assert_same_resolution
-from scilpy.io.utils import assert_inputs_exist
-from scilpy.tracking.tools import subsample_streamlines
+from scilpy.io.utils import (assert_inputs_exist,
+                             add_json_args,
+                             add_overwrite_arg,
+                             add_reference_arg)
 from scilpy.utils.filenames import split_name_with_nii
 from scilpy.utils.metrics_tools import get_metrics_profile_over_streamlines
-
+from scilpy.tracking.tools import (resample_streamlines_num_points,
+                                   resample_streamlines_step_size)
```

Comment:
I think this option makes no sense `resample_streamlines_step_size` the tractprofile has a set number of point, all streamlines need to have the same number of point for it to work, right?

Reply:
Agree